internal/sirius/get_deputy_clients.go

```diff
@@ -196,6 +201,10 @@ func getMostRecentSupervisionLevel(orders []Order) string {
 	sort.Slice(orders, func(i, j int) bool {
 		if orders[i].LatestSupervisionLevel.AppliesFrom == "" {
 			orders[i].LatestSupervisionLevel.AppliesFrom = "01/01/0001"
+
+		}
+		if orders[j].LatestSupervisionLevel.AppliesFrom == "" {
+			orders[j].LatestSupervisionLevel.AppliesFrom = "01/01/0001"
```

Comment:
I might be mistaken, but is this code not doing the same as the below condition?

Reply:
yep thats right also added by mistake